src/modules/account/layouts/PaymentSettingPageLayout/PaymentSettingPageLayout.tsx

```diff
@@ -1,4 +1,4 @@
-import AccountPageHeader from '@modules/account/components/AccountPageHeader/AccountPageHeader';
+import AccountPageHeader from '@modules/account/ui/AccountPageHeader/AccountPageHeader';
```

Comment:
Why separate `ui` and `components`?

Reply:
Same idea in `/src`, reuseable componenet should be in `ui`. In this case, these are reuseable componenets for accountp pages so I moved it into `ui` under the `account` module.

Reply:
I don't think separating them is necessary. We don't really care if a component is reusable or not. It makes searching more difficult, and if we decide this component is now reusable, we have to move it.

Reply:
Hmm, when I said "reuseable", I kinda mean like it is used a lot in this module rather than it's just merely resuseable (since technically, any componenet is reuseable).

For example, I don't want `ActionField` to be in the same directory as `PassowrdActionField` since the former is obviously more general.

Reply:
In the end, we might want to bring them into `/src/ui` because they may be reuseable for the whole app.

Reply:
> For example, I don't want ActionField to be in the same directory as PassowrdActionField since the former is obviously more general.

I don't think this is a problem. The name has already suggested that `PasswordActionField` is built on `ActionField`. How much "general" a component needs to be put inside `/ui`?
 
> In the end, we might want to bring them into /src/ui because they may be reuseable for the whole app.

Not for `ActionField` at least. The /ui suppose to be like a CSS Framework. Only highly general components should be in there.

Reply:
> I don't think this is a problem, or rather, I don't see the benefit of doing this. The name has already suggested that `PasswordActionField` is built on `ActionField`. How much "general" a component needs to be put inside `/ui`?

Just use feelings lol... I'll move them back

Reply:
@Andrewnt219 done